src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -226,6 +226,13 @@ public Task AllowPreflightReuqest()
             RedirectToLeader();
         }
 
+        [RavenAction("/admin/cluster/secede", "GET", "/admin/cluster/secede")]
```

Comment:
Is there a reason to have a dedicated endpoint and not just a script via the admin console?